scalableminds/webknossos
# Adds configurable discussion board link to webKnossos

Adds a link to support.webknossos.org to all webKnossos instances.
It is only shown for admins, because mhlab wants that. Not sure if that makes sense for all labs.

### Steps to test:
- Open webKnossos. Look at the navbar under Help. See documentation and support link.

------
